src/io/flutter/logging/FlutterLogEntryParser.java

```diff
@@ -332,4 +396,9 @@ public FlutterLogEntry parseConsoleEvent(String text, ConsoleViewContentType typ
     // TODO(pq): handle else (errors, etc).
     return null;
   }
+
+  public void setupInspector(@NotNull FlutterApp app, @NotNull VmService vmService) {
```

Comment:
Nit: might be nice to match the naming convention of similar methods that take an app and vmService to initialize debugger state?

Reply:
Sure thing.  Done.